docs/basic_training/processes.md

```diff
@@ -1026,6 +1026,10 @@ workflow {
 
 The `when` declaration allows you to define a condition that must be verified in order to execute the process. This can be any expression that evaluates a boolean value.
 
+!!! warning
+
+    Deprecated since version 24.10.0: Use conditional logic (e.g. `if` statement, ![filter](https://www.nextflow.io/docs/latest/reference/operator.html#operator-filter) operator) in the calling workflow instead.
```

Comment:
```suggestion
    Deprecated since version 24.10.0: Use conditional logic (e.g. `if` statement, [filter](https://www.nextflow.io/docs/latest/reference/operator.html#operator-filter) operator) in the calling workflow instead.
```